newsroom/settings.py

```diff
@@ -109,15 +113,33 @@ def validate_general_settings(values):
                 return gettext("{}: Email IDs not in proper format".format(field_txt))
 
 
+def sync_get_setting(setting_key=None):
+    loop = asyncio.get_event_loop()
```

Comment:
Rather than doing this, it would be good to await the appropriate data first before rendering the template.
Also, does Flask3/Jinja not support async inside templates?

Reply:
yep jinja is not supporting async functions inside templates

Reply:
Actually, I tried different approaches to use these async functions in the templates, but in the end, the issue was solved by this hack